app/javascript/components/ansible-credentials-form/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { API } from '../../http_api';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './ansible-credentials-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const AnsibleCredentialsForm = ({ recordId }) => {
+  const [{ fields, initialValues, isLoading }, setState] = useState({ fields: [], isLoading: !!recordId });
+  const promise = useMemo(() => API.options('/api/authentications'), []);
+  const submitLabel = !!recordId ? __('Save') : __('Add');
+
+  const loadSchema = (value, appendState = {}) => ({
+    data: {
+      credential_types: { embedded_ansible_credential_types },
+    },
+  }) =>
+    setState(state => ({
+      ...state,
+      ...appendState,
+      fields: embedded_ansible_credential_types[value].attributes,
+    }));
+
+    useEffect(() => {
+      API.get('/api/providers?collection_class=ManageIQ::Providers::EmbeddedAutomationManager').then(({ resources: [manager_resource] }) => {
+        if (!recordId) {
+          setState({
+            initialValues: {
```

Comment:
This change is destructive towards the `fields` in the `state`. So it should be:
```
setState((state) => ({
  ...state,
  initialValues: {
    manager_resource,
  },
  isLoading: false,
});
```